File: app/src/main/java/com/example/learntogether_mobile/API/ResponseU.java
package com.example.learntogether_mobile.API;

import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Класс универсальный для обработки/отправки запросов. Сгенерирован скриптами на основе кода для бэкенда,
 * содержимое ответа (значения, которые могут быть не null для данного запроса) см. в коде сервера.
 *
 * Ещё раз, класс СГЕНЕРИРОВАН СКРИПТАМИ, не человеком, разбираться в этом болоте не нужно!
 */
public class ResponseU {
    @SerializedName("Error")
    public String Error;

    @SerializedName("Success")
    public boolean Success;

    @SerializedName("Comments")
    public List<ListU> Comments;

    @SerializedName("Asks")
    public List<ListU> Asks;

    @SerializedName("Infos")
    public List<ListU> Infos;

    @SerializedName("Contents")
    public String Contents;

    @SerializedName("Meetings")
    public List<ListU> Meetings;

    @SerializedName("Responds")
    public List<ListU> Responds;

    @SerializedName("Result")
    public String Result;

    @SerializedName("Roles")
    public List<ListU> Roles;

    @SerializedName("Complaints")
    public List<ListU> Complaints;

    @SerializedName("Token")
    public String Token;

    @SerializedName("List")
    public List<String> stringList;

    @SerializedName("NotificationPort")
    public int NotificationPort;
}

File: app/src/main/java/com/example/learntogether_mobile/API/ResponseU.kt
package com.example.learntogether_mobile.API

import com.google.gson.annotations.SerializedName

/**
 * Класс универсальный для обработки/отправки запросов. Сгенерирован скриптами на основе кода для бэкенда,
 * содержимое ответа (значения, которые могут быть не null для данного запроса) см. в коде сервера.
 *
 * Ещё раз, класс СГЕНЕРИРОВАН СКРИПТАМИ, не человеком, разбираться в этом болоте не нужно!
 */
class ResponseU {
    @JvmField
    @SerializedName("Error")
    var Error: String? = null

    @SerializedName("Success")
    var Success: Boolean? = false

    @SerializedName("Comments")
    var Comments: List<ListU>? = null

    @SerializedName("Asks")
    var Asks: List<ListU>? = null

    @SerializedName("Infos")
    var Infos: List<ListU>? = null

    @SerializedName("news")
    var news: List<ListU>? = null

    @SerializedName("tasks")
    var tasks: List<ListU>? = null

    @SerializedName("votes")
    var votes: List<ListU>? = null

    @SerializedName("Contents")
    var Contents: String? = null

    @SerializedName("Meetings")
    var Meetings: List<ListU>? = null

    @SerializedName("Responds")
    var Responds: List<ListU>? = null

    @JvmField
    @SerializedName("Result")
    var Result: String? = null

    @SerializedName("Roles")
    var Roles: List<ListU>? = null

    @SerializedName("Complaints")
    var Complaints: List<ListU>? = null

    @JvmField
    @SerializedName("Token")
    var Token: String? = null

    @SerializedName("List")
    var stringList: List<String>? = null

    @SerializedName("NotificationPort")
    var NotificationPort: Int? = 0

    @JvmField
    @SerializedName("users")
    var users: List<ListU>? = null
}
